src/pages/contributors/alexander-brandes.adoc

```diff
@@ -0,0 +1,81 @@
+Everything I needed to learn, I had to teach myself since I had no technical background at all.
+The knowledge acquired helped me tremendously when getting into open source and development. 
+I still enjoy looking back at what started as a small project and turned into something thousands of people have a great time on every day.
+It is absolutely inspiring for me to see.
+
+Before I started contributing to Jenkins, I had already contributed to smaller open-source projects.
+My contributions primarily focused on Java-based projects and documentation improvements.
+These contributions, paired with continuous learning of new languages and technologies, helped improve my knowledge and advance future contributions.
+
+== How long have you been using Jenkins?
+
+I have been using Jenkins since 2019.
+
+== Why choose Jenkins over other projects?
+
+Jenkins maintains a notably low barrier of entry, which makes it attractive to contribute to, regardless of the massive plugin ecosystem around Jenkins.
+What I liked the most when I started contributing to Jenkins, is that you can start with small pull requests for plugins, and move on to bigger contributions when you feel more comfortable with the codebase and ecosystem.
+
+Moreover, Jenkins is something I had used before I started contributing.
+Therefore, I already had a certain level of experience as an end user, before becoming a contributor.
+That's a key point which made it easier for me to contribute, because I've already been a user of the project, instead of a contributing to a project totally unfamiliar to me.
+
+== What problems has Jenkins solved for you?
+
+When I started using Jenkins initially, its primary task was only building a project.
+Over the years, my needs shifted and so did my setup.
+Nowadays, Jenkins is my all-rounder, handling a multitude of tasks.
+
+I'm using various plugins to deploy artifacts to a Maven repository, aggregate test results and code coverage, and send notifications to Discord when a new build is available.
+Additionally, Jenkins has helped me to test pull requests before they land in the default branch.
+
+Jenkins has helped me automate tasks, allowing me to spend my time and energy on more important tasks.
+
+== Is there an aspect of Jenkins that you're particularly passionate about?
+
+The Jenkins project has existed for almost a decade, involving components created around that time that are still being maintained and developed.
+Jenkins is great at reforging itself, especially in code.
+Constant UI/UX improvements over the years is great to see.
+
+The connection with the community is a keypoint when it comes to running a successful open source project.
+Hence, I value the ease of accessibility of the project's chat channels and community forums a lot.
+I greatly appreciate the opportunity to connect with other members of the community with such ease.
```

Comment:
Repetition of 'ease'. 
Maybe 'I greatly appreciate the opportunity to connect with other members of the community with such simplicty.'?
or 'I greatly appreciate the opportunity to readily connect with other members of the community.'?

Reply:
Thanks @liamfield, I was having trouble trying to figure out the best way to rephrase that so the repetition isn't there.